runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.hpp

```diff
@@ -106,6 +106,31 @@ class ARM64PrivateLinkage : public TR::Linkage
     */
    virtual void createEpilogue(TR::Instruction *cursor);
 
+   /**
+    * @brief Loads all parameters passed on the stack from the interpreter into
+    *        the corresponding JITed method private linkage register.
+    *
+    * @param[in] cursor : the TR::Instruction to begin generating
+    *        the load sequence at.
+    *
+    * @return : the instruction cursor after the load sequence
+    */
+   TR::Instruction *loadStackParametersToLinkageRegisters(TR::Instruction *cursor);
+
+   /**
+    *
+    * @brief Copies parameters from where they enter the method (either on stack or
+    *        in a linkage register) to their "home location" where the method body
+    *        will expect to find them (either on stack or in a global register).
```

Comment:
This function only seems to handle register parameters but the comment suggests that it does (or should do) more. Are stack parameters already in their home locations in this linkage or is there something else amiss?

Reply:
This is an example of me getting ahead of myself.  The ultimate purpose of this function is to do what the comment says, but the current implementation is much more limited than that.  At the moment, GRA is not supported on AArch64.  When it is, this function will be responsible for shuffling parameters passed in linkage registers into GRA registers (if they were selected) or spilling them to the stack (if they weren't).  Also, if a parameter was passed in a stack location was selected as a GRA candidate then this function will load from the stack into the appropriate GRA register.

Since, at the moment, all this function does is store the parameters passed in registers to the stack, I've update the comment to reflect that.